gradio/blocks.py

```diff
@@ -1694,15 +1694,20 @@ def attach_load_events(self):
                 ):
                     load_fn, every = component.load_event_to_attach
                     # Use set_event_trigger to avoid ambiguity between load class/instance method
-                    self.set_event_trigger(
+                    dep = self.set_event_trigger(
                         "load",
                         load_fn,
                         None,
                         component,
                         no_target=True,
-                        queue=False,
```

Comment:
qq: why was slider randomization added to the queue in the original issue if `queue=False`?

Reply:
They were being added by this method https://github.com/gradio-app/gradio/blob/d906dedf97eb265f1d3b4df20188a1c7ee617f76/gradio/components.py#L224 instead which did not specify whether it should be queued so the top-level enable_queue parameter took precedence (True on spaces)

Reply:
Ah now it makes sense, thank you!